tick/hawkes/inference/hawkes_cumulant_matching.py

```diff
@@ -279,6 +279,7 @@ def _tf_placeholders(self):
         d = self.n_nodes
         if self._tf_feed_dict is None:
             with self._tf_graph.as_default():
+                tf.compat.v1.disable_eager_execution()
```

Comment:
Along with this the `tf.graph` initialization might also require a change:
https://github.com/X-DataInitiative/tick/blob/master/tick/hawkes/inference/hawkes_cumulant_matching.py#L166 
As listed there: https://www.tensorflow.org/guide/migrate#top-level_behavioral_changes

Reply:
that line that wasn't meant to be commit and has been removed

Reply:
that line wasn't meant to be commit and has been removed